src/IRTS/Compiler.hs

```diff
@@ -108,7 +110,7 @@ compile codegen f mtm
             Just f -> runIO $ writeFile f (dumpDefuns defuns)
         triple <- Idris.AbsSyntax.targetTriple
         cpu <- Idris.AbsSyntax.targetCPU
-        logCodeGen 1 "Building output"
+        when v $ iputStrLn "Generating Code."
```

Comment:
Yes, I am experimenting with getting helpful verbose output to track progress through the compiler. I could always reinstate the logging output as well, but that seems a bit too verbose when logging is turned on.